tests/capture/test_capture_cond.py

```diff
@@ -733,6 +737,7 @@ def f(*x):
 
         assert np.allclose(res, expected, atol=atol, rtol=0), f"Expected {expected}, but got {res}"
 
+    @pytest.mark.xfail(strict=False)
```

Comment:
I found the comments above on why `strict=False` very helpful. Mind adding one here as well? :)